packages/element-lib/test/Truffle-Tests/Element Contract Test Explainer.md

```diff
@@ -0,0 +1,90 @@
+# Element Contract Test Explainer
+
+Primary Modules:
+Ganache CLI v6.9.0 (ganache-core: 2.10.1)
+Truffle v5.1.36 (core: 5.1.36)
+Solidity v0.5.16 (solc-js)
+Node v12.14.0
+Web3.js v1.2.1
+Chai 4.2.0
+Truffle Assertions 0.9.2
+Ganache Time Traveler 1.0.14
+
+To begin, clone the repo.
+
+Then start ganache from your node command line window in the element folder:
```

Comment:
Specify the full path of the folder (element/packages/element-lib/test/Truffle-Tests/element) , as "element" is also the name of the root level directory.

Reply:
sorry, overlooked will rename to element-contract-test

Reply:
done with new commit @gjgd